Objective-C/CBLMutableDocument.h

```diff
@@ -110,6 +110,12 @@ NS_ASSUME_NONNULL_BEGIN
                                 json: (NSString*)json
                                error: (NSError**)error;
 
+/**
+ Returns a mutable copy of the document.
+ 
+ @return The CBLMutableDocument object.
+ */
+- (CBLMutableDocument*) copy;
```

Comment:
I might be wrong as it has been awhile. I don't remember if we need to explicity define this as the copy is a part of NSObject protocol.